lib/routes.js

```diff
@@ -24,7 +24,8 @@ const defaults = {
 		national:    true,
 		regionalExp: true,
 		regional:    true
-	}
+	},
+	bahncard: 		0 //25_1: 1, 25_2: 2, 50_1: 3, 50_2: 4
```

Comment:
🎉 Turns out @juliuste has already figured them out: https://gist.github.com/juliuste/202bb04f450a79f8fa12a2ec3abcd72d

@RomanKarwacik you might also want to have a look at the requests we recorded from DB Navigator: https://gist.github.com/derhuerst/2a735268bd82a0a6779633f15dceba33